cockatrice/src/shortcutssettings.cpp

```diff
@@ -114,8 +113,15 @@ bool ShortcutsSettings::isValid(QString name, QString Sequences)
 
     QString checkSequence = Sequences.split(";").last();
 
-    QStringList forbiddenKeys = (QStringList() << "Del" << "Down" << "Up" << "Left" << "Right"
-        << "Return" << "Enter" << "Backspace" << "Esc");
+    QStringList forbiddenKeys = (QStringList() << "Del"
+                                               << "Down"
+                                               << "Up"
+                                               << "Left"
+                                               << "Right"
+                                               << "Return"
+                                               << "Enter"
+                                               << "Backspace"
+                                               << "Esc");
```

Comment:
What about `CapsLock` and `Menu` (or application) keys on windows?
No idea about mac...

Reply:
Good idea, haven't thought about those!
Could we make a list of all such keys so that I could insert them in one commit?

Reply:
Numlock and scrolllock should also be banned imo

Reply:
`Print` and `Pause` ?

Reply:
If the user would like to use numlock or pause why should we forbid it?
I think this ban should only include keys that actually break the software in some way, e.g. shortcut on arrow keys would make tree navigation impossible with keyboard; `menu` key would pop up a menu whenever used; `del` would (and did!) prevent emitting some signals, etc.
Other keys that are just simply inconvenient should be fair game in my opinion.